crates/glaredb/src/args/local.rs

```diff
@@ -38,6 +38,9 @@ pub struct LocalClientOpts {
     #[clap(flatten)]
     pub storage_config: StorageConfigArgs,
 
+    #[clap(long, default_value = "false", hide = true)]
+    pub timing: bool,
```

Comment:
This could also not be hidden, but I didn't feel like it was worth extending the existing cli args just with this.